src/styles/style.css

```diff
@@ -132,22 +133,39 @@ body {
 
 /* Home Tab */
 
-.invest-list-table {
-	display: block;
+.invest-list-group {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
 	height: 87vh;
 	overflow-y: auto;
 }
 
-.invest-list-table .table td {
+.invest-list-group .list-group-item {
+	width: auto;
 	border-top: none;
-	border-bottom: 1px solid #dee2e6;
+	border-right: none;
+	border-left: none;
+	border-bottom: 1px solid rgba(0,0,0,.125);
```

Comment:
Just out of curiosity, are RGBA colors preferred over hex?

Reply:
I really don't know. The diff makes it looks like that was the main point of this change, but that's just a coincidence. I had purged all the old rules before creating the new ones.

Reply:
This made me wonder about representing alpha (transparency) in hex, and sure enough you can tack on a leading two element marker to represent transparency. Fun!
<html>
<body>
<!--StartFragment-->

Alpha Level | HEX Code
-- | --
100% | FF
95% | F2
90% | E6
85% | D9
80% | CC
75% | BF
70% | B3
65% | A6
60% | 99
55% | 8C
50% | 80
45% | 73
40% | 66
35% | 59
30% | 4D
25% | 40
20% | 33
15% | 26
10% | 1A
5% | 0D
0% | 00

<!--EndFragment-->
</body>
</html>